lib/cs_guide/resources/venue.ex

```diff
@@ -74,7 +75,7 @@ defmodule CsGuide.Resources.Venue do
     |> Map.put(:updated_at, nil)
     |> __MODULE__.changeset(attrs)
     |> Resources.put_many_to_many_assoc(attrs, :venue_types, CsGuide.Categories.VenueType, :name)
-    |> Resources.put_many_to_many_assoc(attrs, :drinks, CsGuide.Resources.Drink, :name)
+    |> Resources.put_many_to_many_assoc(attrs, :drinks, CsGuide.Resources.Drink, :entry_id)
     |> Repo.insert()
```

Comment:
how are we _sanitising_ data before it gets inserted into the DB to avoid XSS?
see: https://github.com/dwyl/fields/issues/19